x/evm/keeper/fees_test.go

```diff
@@ -309,16 +320,18 @@ func (suite *KeeperTestSuite) TestVerifyFeeAndDeductTxCostsFromUserBalance() {
 			gasLimit:         100,
 			gasPrice:         &oneInt,
 			cost:             &oneInt,
+			initialBalance:   initBalance,
 			accessList:       &ethtypes.AccessList{},
 			expectPassVerify: true,
 			expectPassDeduct: true,
 			from:             suite.address.String(),
 		},
 		{
 			name:             "Higher gas limit, not enough balance",
-			gasLimit:         1000000,
+			gasLimit:         105,
```

Comment:
Does this 105 depends on the initialBalance var?

Reply:
yes, is related to it